<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="gameLayout_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>427</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gameLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="definitionWidget">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJeu">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_partie"/>
    <addaction name="actionCharger_une_partie"/>
    <addaction name="actionEnregistrer_partie"/>
    <addaction name="actionExporter_grille"/>
    <addaction name="exportPDF"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuPartie">
    <property name="title">
     <string>Jeu</string>
    </property>
    <addaction name="actionV_rifier_la_grille"/>
    <addaction name="actionCorriger_la_grille"/>
    <addaction name="actionVider_la_grille"/>
   </widget>
   <widget class="QMenu" name="menuDictionnaires">
    <property name="title">
     <string>Dictionnaires</string>
    </property>
    <addaction name="actionGestion_des_dictionnaire"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionMenu_Aide"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuJeu"/>
   <addaction name="menuPartie"/>
   <addaction name="menuDictionnaires"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouvelle_partie">
   <property name="text">
    <string>Nouvelle partie</string>
   </property>
  </action>
  <action name="actionCharger_une_partie">
   <property name="text">
    <string>Charger une partie</string>
   </property>
  </action>
  <action name="actionEnregistrer_partie">
   <property name="text">
    <string>Enregistrer partie</string>
   </property>
  </action>
  <action name="actionExporter_grille">
   <property name="text">
    <string>Exporter grille</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCorriger_la_grille">
   <property name="text">
    <string>Corriger la grille</string>
   </property>
  </action>
  <action name="actionVider_la_grille">
   <property name="text">
    <string>Vider la grille</string>
   </property>
  </action>
  <action name="actionGestion_des_dictionnaire">
   <property name="text">
    <string>Gestion des dictionnaire</string>
   </property>
  </action>
  <action name="actionMenu_Aide">
   <property name="text">
    <string>Menu Aide</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionV_rifier_la_grille">
   <property name="text">
    <string>VÃ©rifier la grille</string>
   </property>
  </action>
  <action name="exportPDF">
   <property name="text">
    <string>Export PDF</string>
   </property>
   <property name="toolTip">
    <string>Exporter en PDF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
